packages/integration_test/android/src/test/java/dev/flutter/plugins/integration_test/FlutterDeviceScreenshotTest.java

```diff
@@ -12,14 +12,18 @@
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
+import org.junit.runner.RunWith;
+
 import android.app.Activity;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
 
 import io.flutter.embedding.android.FlutterActivity;
 import io.flutter.embedding.android.FlutterFragment;
 import io.flutter.embedding.android.FlutterFragmentActivity;
 import io.flutter.embedding.android.FlutterView;
 
-public class FlutterDeviceScreenshotTest extends AndroidJUnitRunner {
```

Comment:
nit: can we remove the import of `AndroidJUnitRunner` too?

Reply:
Yes